reading/phoenix_authentication.livemd

```diff
@@ -243,7 +249,7 @@ R --> C --> V --> T
 
 <!-- livebook:{"break_markdown":true} -->
 
-Submit the form with any email (real or fake such as `test@test.com`) and any valid password such as `supersecretpassword`. We should never re-use personal passwords, particularly during development where they may not be private.
+Submit the form with any email (real or fake such as `test@example.com`) and any valid password such as `supersecretpassword`. We should never re-use personal passwords, particularly during development where they may not be private.
```

Comment:
"test . com" is a real, live domain that shouldn't be used for any kind of testing (or anything else, by the looks of it). example.com, and some other domains, are reserved by the IANA for testing. https://en.wikipedia.org/wiki/Example.com

Reply:
Thank you! I did not know!